controllers/specialresource.go

```diff
@@ -89,7 +89,9 @@ func SpecialResourcesReconcile(r *SpecialResourceReconciler, req ctrl.Request) (
 	log.Info("Resolving Dependencies")
 
 	pchart, err := helmer.Load(r.parent.Spec.Chart)
-	exit.OnError(err)
+	if err != nil {
```

Comment:
```suggestion
	if err != nil {
		operatorStatusUpdate(&r.parent, fmt.Sprintf("%v", err))
```
We want to add the error to the CR status here

Reply:
I still think this is needed to set CR status about the failed helmer.Load @zvonkok

Reply:
Fixed.